### PR TITLE
Add Linpuss token to Linea token shortlist

### PR DESCRIPTION
**Pull request by ntbenti**
Action: add
Context / rational: we have created a memecoin which is doing quite well and Christopher advised me to add the token here to enable tracking in the metamask wallet. We are already listed on coingecko for price tracking.

PR checklist:
- [x] I've verified and published the contract source
- [x] I've kept the token list in alphabetical order based on token symbol
- [x] I've updated the `updatedAt` (and potentially `createdAt`) fields
- [x] I've updated the list version number as per README instruction

### CODE REVIEW

**Review by thedarkjester — CHANGES_REQUESTED** [review]
Please also bump the minor version and updatedAt date here:
https://github.com/Consensys/linea-token-list/pull/188/files#diff-aaa9bed4b5e7293d486d6c466c4ce400831759b3b1e97084c4f49d23e61277a7R5-R12

File: json/linea-mainnet-token-shortlist.json [hunk 1]
```diff
@@ -374,6 +374,21 @@
...
+      "tokenId": "https://lineascan.build/address/0x81BE2aCB2e9291dB6400f9f6a4D0f35f24de2E77",
+      "tokenType": [
+        "native"
+      ],
+      "address": "0x81BE2aCB2e9291dB6400f9f6a4D0f35f24de2E77",
+      "name": "Linpuss",
+      "symbol": "LPUSS",
+      "decimals": 18,
+      "createdAt": "2024-11-20",
+      "updatedAt": "2024-11-20",
```
[the remark this answers is not in the document]
Comment (ntbenti) [reply json/linea-mainnet-token-shortlist.json:389]: would u be able to update for me please, date is 2024-10-31 for both. Thanks

[the remark this answers is not in the document]
Comment (thedarkjester) [reply json/linea-mainnet-token-shortlist.json:389]: you should be able to do it on your fork and push - would come through correctly.


Don't forget the minor version and major updated date.

**Review by thedarkjester — APPROVED** [review]

**Review by VGau — APPROVED** [review]

**Comment by ntbenti** [comment]
Hey, is there something wrong from my end? 
If not, how long should it take to show in metamask?

**Pull request merged**